.licences/overrides/overrides.json

```diff
@@ -0,0 +1 @@
+{"name": "github.com/bmizerany/assert", "licenceType" : "MIT" }
```

Comment:
maybe we could phase out this package in the future, since it's not maintained anymore and it seems that the functionality exists in go anyways?